src/main/java/com/linkedin/replica/wall/commands/impl/AddPostCommand.java

```diff
@@ -56,6 +60,7 @@ public Object execute() throws ParseException {
         post = new Post(authorId, type, companyId, privacy, text, hashtags, mentions, likesCount, images, videos, urls, commentsCount, timestamp, isCompanyPost, isPrior, shares, headLine,isArticle);
 
         String response = dbHandler.addPost(post);
```

Comment:
What form of response could be returned ?

Reply:
Either post is successfully added or not

Reply:
I think boolean response is better, what do you think?